modules/kafka/kafka.go

```diff
@@ -15,20 +15,37 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 )
 
-const publicPort = nat.Port("9093/tcp")
+const (
+	// Mapped port for advertised listener of localhost:<mapped_port>.
+	PublicLocalhostPort = nat.Port("9093/tcp")
+	// Mapped port for advertised listener of host.docker.internal:<mapped_port>
+	PublicDockerHostPort = nat.Port("19092/tcp")
+	// Listening port for advertised listener of <container_name>:19093. This is not mapped to a random host port
+	NetworkInternalContainerNamePort = 19093
+	// Listening port for advertised listener of <container_id>:19094. This is not mapped to a random host port
+	NetworkInternalContainerIdPort = 19094
+	// Listening port for Broker intercommunication
+	BrokerToBrokerPort = 9092
+	// Listening port for Contoller
+	ControllerPort = 9094
+)
+
 const (
 	starterScript = "/usr/sbin/testcontainers_start.sh"
 
 	// starterScript {
-	starterScriptContent = `#!/bin/bash
+	starterScriptContent = `
+#!/bin/bash
```

Comment:
bug: scripts shouldn't have a leading newline.

Reply:
Ah, good point. You cool with leaving the raw string with the leading newline but then running `strings.TrimLeft(script)` ?

Reply:
I wouldn't if you want this to be more readable I'd be tempted to move it to a file and include with embed.